scripts/encrypt_private_key.ts

```diff
@@ -0,0 +1,46 @@
+import { command, run, string, positional, option, } from 'cmd-ts';
+import * as fs from 'fs';
+
+const wallet = require("ethereumjs-wallet").default
+
+const app = command({
+  name: 'encrypt_private_key',
+  args: {
+    private_key: positional({
+      type: string,
+      displayName: 'private_key',
+      description: "Your ethereum private key.",
+    }),
+    password: positional({
+      type: string,
+      displayName: 'password',
+      description: "Password used to decrypt encrypted private key.",
+    }),
+    keyfile_name: option({
+      type: string,
+      description: "Name of encrypted JSON file",
+      defaultValue: () => null,
+      short: 'k',
+      long: 'keyfile_name',
+    }),
+  },
+  handler: async ({ private_key, password, keyfile_name }) => {
+    const privateKeyAsUint8Array = Buffer.from(private_key, 'hex');

```

Comment:
Actually from a security perspective not so sure this is necessary.

Say I'm a committee member. I would imagine the flow would go something like:

1. I run the `scripts/encrypt_private_key.ts` command locally on my own machine and generate `akilesh.keyfile.json`, which is the encrypted JSON file.
2. I then push `akilesh.keyfile.json` to the cloud. 
3. I add my `.env` file with my `KEYFILE_PASSWORD` in the cloud.
4. I can now run all operations in the cloud.

At no point am I sending my private key outside my local machine...even if it is a CLI arg to the `scripts/encrypt_private_key.ts` command.

Let me know your thoughts.

Reply:
Yes, the password never leaves the local machine.  My concern is that the private key is entered on the command line, which I think is bad practice as it ends up in the shell history unless the user takes steps to explicitly avoid that.  So I think it would be better to have it in a private key in a file, or pasted through stdin.  I realise this workflow is probably only for dummy cases, but I think most admins will expect this and it will stand out if we don't do it.

Reply:
Cool cool will add it in.